llvm/lib/Target/AArch64/AArch64InstrInfo.cpp

```diff
@@ -7909,7 +7909,7 @@ AArch64InstrInfo::getOutlinableRanges(MachineBasicBlock &MBB,
       CreateNewRangeStartingAt(MI.getIterator());
       continue;
     }
-    LRAvailableEverywhere &= LRU.available(AArch64::LR);
```

Comment:
I think the positive is available check is more readable than having to negate most of the uses